src/packages/core/api/balance.ts

```diff
@@ -0,0 +1,31 @@
+import * as config from 'config';
```

Comment:
Please, create a /api/account.ts file instead where we will place all the account api related features. (Try to group things by entity, not by functionality)

Reply:
Updated in Latest commit